libs/Microsoft.MixedReality.WebRTC/Interop/RemoteAudioTrackInterop.cs

```diff
@@ -2,18 +2,22 @@
 // Licensed under the MIT License.
 
 using System;
-using System.Diagnostics;
 using System.Runtime.InteropServices;
 using static Microsoft.MixedReality.WebRTC.Interop.AudioTrackInterop;
 
 namespace Microsoft.MixedReality.WebRTC.Interop
 {
     internal class RemoteAudioTrackInterop
     {
-        internal class ReadBufferHandle : SafeHandle
+        internal sealed class RemoteAudioTrackHandle : ObjectHandle
```

Comment:
nit: not strictly related, but we have a lot of `MyLongClassNameInterop.MyLongClassName_DoThis` and here even `MyLongClassNameInterop.MyLongClassNameHandle`. It would be nice to have the class name just once in the qualified name.

Reply:
Will leave as a clean-up for `master`.